CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 * `TabModel` now supports a bindable `tooltip`, which can be used to render strings or elements
   while hovering over tabs.
 * New `Placeholder` component provides a thin wrapper around `Box` with standardized, muted styling.
+* `Select` now implements more intelligent filtering of options based on keyboard input.  Filtering
```

Comment:
"more intelligent" is perhaps too subjective for a document that strives to be factual?